spec/javascripts/models/beta/sprint_spec.js

```diff
@@ -0,0 +1,66 @@
+import * as Sprint from 'models/beta/sprint';
+
+describe('Sprint model', () => {
+  const doneSprint = { hasStories: '' }
+  const undoneSprint = {}
+
+  describe('sort', () => {
+    describe('when is done sprints', () => {
+      const amountOfSprints = [1,10,100]
+
+      amountOfSprints.forEach(amount => {
```

Comment:
What is the purpose of this?
I imagine you want to test these follow scenarios:
- When all sprints are done
- When not all sprints are done
- When none sprints are done

So, I would rewrite like this:
```js
descrive('sort', () => {
  describe('When all sprints are done', () => {
    const sprints = [
      { id: 1, hasStories: '' },
      { id: 2, hasStories: '' },
      { id: 3, hasStories: '' },
    ];

    it('returns the sprints list reversed', () => {
      expect(Sprint.sort(sprints)).toEqual([
        { id: 3, hasStories: '' },
        { id: 2, hasStories: '' },
        { id: 1, hasStories: '' },
      ])
    });
  });

  describe('When not all sprints are done', () => {
    const sprints = [
      { id: 1, hasStories: '' },
      { id: 2, hasStories: '' },
      { id: 3 },
    ];

    it('returns the sprints list', () => {
      expect(Sprint.sort(sprints)).toEqual([
        { id: 1, hasStories: '' },
        { id: 2, hasStories: '' },
        { id: 3 },
      ])
    });
  });

  describe('When none sprints are done', () => {
    const sprints = [
      { id: 1 },
      { id: 2 },
      { id: 3 },
    ];

    it('returns the sprints list', () => {
      expect(Sprint.sort(sprints)).toEqual([
        { id: 1 },
        { id: 2 },
        { id: 3 },
      ])
    });
  });
});```

Reply:
I didn't execute the sort function into "beforeEach" because does not have set up and we don't need to test multiple things (more than one `it`).

Reply:
I also use `toEqual` because that way I know for sure what I should expect of this method.

Reply:
Right, I did that `forEach` just to test when have more sprints, but I had not thought in this cases.